tool/tctl/main.go

```diff
@@ -42,6 +42,7 @@ func main() {
 		&common.LockCommand{},
 		&common.BotsCommand{},
 		&common.InventoryCommand{},
+		&common.SessionsCommand{},
```

Comment:
I think you need to add this to enterprise's tctl also: https://github.com/gravitational/teleport.e/blob/master/tool/tctl/main.go.

Reply:
Should that be done in this PR or should I make a separate PR for the change to the enterprise submodule